src/framework/components/collision/component.js

```diff
@@ -62,6 +62,7 @@ Object.assign(pc, function () {
      * <li><strong>mesh</strong>: A collision volume that uses a model asset as its shape.</li>
      * <li><strong>sphere</strong>: A sphere-shaped collision volume.</li>
      * </ul>
+     * @property {pc.Vec3} offset The offset that the component position have from its entity position.
```

Comment:
I'd say `The local space offset of the collision component in relation to its owner entity.`.